install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -1,5 +1,30 @@
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
+metadata:
+  name: machine-config-controller
+  namespace: openshift-machine-config-operator
+  labels:
+    k8s-app: machine-config-controller
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+spec:
+  groups:
+    - name: mcc-important-config-paosed
+      rules:
+        - alert: MCCImportantConfigPaused
+          expr: |
+             mcc_important_config_paused > 0
+          labels:
+            severity: warning
```

Comment:
Firing `warning` alerts immediately seems like it's wound a bit too tightly.  We can go a while before the lack of certificate rotation becomes a problem (significant fractions of a year?).  So setting a large-ish `for` here seems useful to avoid desensitizing folks by alerting every time they briefly pause.  Of course, you could have someone leaving pools paused for the bulk of that time, and then very briefly unpause, have Prom happen to scrape, and then pause again, before the pool had time to rotate out many certs.  Which is one reason "are we paused now, and have we been paused for a bit?" only a rough proxy for "are we at risk of certs expiring?".  And that makes picking an appropriate `for` difficult.  But if we stick with the paused-pool angle on this, I'd expect at least `for: 60m` on this alert, based on [our consistency guidelines][1].

[1]: https://github.com/openshift/enhancements/blob/15aebc7549095cfc7eebc26d6460defc8e3927f6/enhancements/monitoring/alerting-consistency.md#warning-alerts